src/pretalx/submission/models/review.py

```diff
@@ -42,9 +43,29 @@ def recalculate_scores(cls, event):
 
     def save(self, *args, **kwargs):
         if self.is_independent:
+            if (
+                not ReviewScoreCategory.objects.exclude(pk=self.pk)
+                .filter(is_independent=False, event=self.event)
+                .exists()
+            ):
+                raise ValidationError(
+                    _("Atleast one non-independent score category is required."),
+                )
             self.weight = 0
         return super().save(*args, **kwargs)
 
+    def delete(self):
+        if (
+            self.is_independent
```

Comment:
I think `self.is_independent` because if you have two reviews

1. One Non-Independent - score
2. One Independent - grammer

If you delete `score` this would let you delete that review score since it's not an independent review hence does not got inside the check.